runtime/include/arch/aarch64/context.h

```diff
@@ -46,25 +45,22 @@ arch_context_switch(struct arch_context *a, struct arch_context *b)
 	/*
 	 * Assumption: In the case of a slow context switch, the caller
 	 * set current_sandbox to the sandbox containing the target context
+	 * Temporally comment because it will cause compilation error
 	 */
-	if (b->variant == ARCH_CONTEXT_VARIANT_SLOW) {
+	/*if (b->variant == ARCH_CONTEXT_VARIANT_SLOW) {
 		struct sandbox *current = current_sandbox_get();
 		assert(current != NULL && b == &current->ctxt);
-	}
+	}*/
```

Comment:
Remind me, was this problematic because it caused an include cycle or did we actually see this fail?

Reply:
If this was just because of the header, it probably makes sense to delete this whole `#ifndef NDEBUG` block